### PR TITLE
[Docathon] Fix NO.12，27 API label

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Docs
### Description

* https://github.com/PaddlePaddle/Paddle/issues/57363
No.12
No.27

### CODE REVIEW

**Review by Liyulingyue — DISMISSED** [review]

File: python/paddle/base/layers/layer_function_generator.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def func(x, name=None):
     func.__name__ = inplace_op_type
     func.__doc__ = """
 Inplace version of ``{}`` API, the output Tensor will be inplaced with input ``x``.
-Please refer to :ref:`api_base_layers_{}`.
+Please refer to :ref:`api_paddle_layers_{}`.
```
Comment (sunzhongkai588) [inline python/paddle/base/layers/layer_function_generator.py]:
```suggestion
Please refer to :ref:`api_paddle_base_layers_{}`.
```

File: python/paddle/base/lod_tensor.py [hunk 2]
```diff
@@ -128,7 +128,7 @@ def create_random_int_lodtensor(
        :code:`base_shape` .
 
     2. Create a numpy array of random integers, and parse the created numpy
-       array as parameter :code:`data` of :ref:`api_base_create_lod_tensor` to
+       array as parameter :code:`data` of :ref:`api_paddle_create_lod_tensor` to
```
Comment (sunzhongkai588) [inline python/paddle/base/lod_tensor.py]:
```suggestion
       array as parameter :code:`data` of :ref:`api_paddle_base_create_lod_tensor` to
```

**Review by sunzhongkai588 — COMMENTED** [review]
其实都是非公开的API，不过也改成符合规范的ref

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**